### DISCUSSION TITLE
v2 route file naming

### DISCUSSION THREAD

**Discussion by Joseamica** [remix-run/remix#6592] [Q&A]
My app is growing as files and I need to make a folder structure.

Reading the documentation, it's not clear enough for me so I want to make sure Im doing it correctly.

I want to make an admin section, but folder organized.

I tried this approach:

```
routes/
  /admin
     /route.tsx (outlet inside)
     /testing.tsx -> No routes matched location "/admin/testing"
                              No routes matched location "/testing"
```
  
 What I think its the correct way of doing it is doing now a /admin/testing folder and inside a route.tsx, but seems unconventional

**Comment by kiliman** [comment]
With `v2` routing, you can't use folders for organization. You can only use them for co-location. You can check out `remix-flat-routes`, which supports _hybrid_ routes, that allow you to add folders as needed.
